app/build.gradle

```diff
@@ -130,24 +130,19 @@ dependencies {
 
     implementation 'com.github.jorgecastilloprz:fabprogresscircle:1.01@aar'
     implementation "com.leinardi.android:speed-dial:3.0.0"
-
+    implementation 'org.greenrobot:eventbus:3.1.1'
```

Comment:
what do you use this for? Did you add it in the about section text?

Reply:
As I explained to you earlier, this is for cross-code sending informations through a « bus ». It is pretty useful for sending resquests to another class that is not linked to the current one..

(Like refreshing after upload or album creation..)

Reply:
yes, I've seen (and commented) later. I think it would be better to replace its usage by using the ViewModels...